home_work_7/mod_init.py

```diff
@@ -0,0 +1,84 @@
+import mod_menu
+import mod_add
+import mod_remove
+import mod_remove_all
+import mod_search
+import mod_view
+import json
+import os
+from mod_log import LOG
+from mod_city import get_city_id
+
+
+def get_dir():
+    return os.path.dirname(os.path.realpath(__file__))
+
+@LOG
+def open_exist():
+    with open(get_dir()+'\db_main.json', 'r', encoding='utf-8') as file:
```

Comment:
Мне кажется так чуточку проще и не нужно использовать вашу функцию `get_dir`
```suggestion
    with open(os.path.join(os.getcwd(), 'db_main.json'), 'r', encoding='utf-8') as file:
```